code/game/gamemodes/objective.dm

```diff
@@ -337,9 +337,9 @@ datum/objective/hijack
 		if(issilicon(owner.current))
 			return 0
 		var/area/shuttle = locate(/area/shuttle/escape/centcom)
-		var/list/protected_mobs = list(/mob/living/silicon/ai, /mob/living/silicon/pai)
 		for(var/mob/living/player in player_list)
-			if(player.type in protected_mobs)	continue
+			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role)

```

Comment:
Most technically yes, I'm not too worried about the latter---and the former is soooo is going to be so incredibly rare, that I'm not sure it's worth checking for, considering ERT WILL gun down someone who is trying to kill everyone.